R/int_pctl.R

```diff
@@ -84,6 +84,10 @@ int_pctl.tune_results <- function(.data, metrics = NULL, times = 1001,
 
   if (is.null(metrics)) {
     metrics <- .get_tune_metrics(.data)
+  } else {
+    if (!inherits(metrics, "metric_set")) {
+      cli::cli_abort("{.arg metrics} should be a {.pkg yardstick} {.fun metric_set}.")
```

Comment:
```suggestion
      cli::cli_abort("{.arg metrics} should be a metric set as generated by {.fun yardstick::metric_set}.")
```
Namespacing the function makes the error message clickable in RStudio which I think is pretty neat.